embedx/embedx_test.go

```diff
@@ -0,0 +1,69 @@
+package embedx
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/ory/jsonschema/v3"
+)
+
+func TestAddSchemaResources(t *testing.T) {
+
+	t.Run("case=validate schema resources are loading schema id correctly", func(t *testing.T) {
+		c := jsonschema.NewCompiler()
+		assert.NoError(t, AddSchemaResources(c, Config, IdentityMeta, IdentityExtension))
+
```

Comment:
Loading is not required in this test right?
```suggestion
```

Reply:
yeah it wasn't. It used to be required before i refactored embedx.
I have changed it now